packages/blockchain-wallet-v4-frontend/src/modals/Settings/RecoveryPhrase/RecoveryPhraseIntro/index.tsx

```diff
@@ -34,6 +34,17 @@ const ButtonWrapper = styled(FlyoutWrapper)`
   }
 `
 
+const ImageContainer = styled.div`
+  @media only screen and (max-height: 600px) {
```

Comment:
This should not be allowed by style-lint

Reply:
it didn't break nor stopped of committing, I can move it to service/ResponsiveService.

Reply:
Yes, and please add @media to stylelint so we avoid this in the future.

Reply:
never mind will update it here